insights/parsers/engine_db_query.py

```diff
@@ -44,6 +46,14 @@ class EngineDBQueryVDSMversion(CommandParser, JSONParser):
         >>> output.result == [{'vds_name': 'hosto', 'rpm_version': 'vdsm-4.30.40-1.el7ev'}]
         True
     """
+    def parse_content(self, content):
+        if not content:
+            raise SkipException("Empty output.")
+        try:
+            self.data = json.loads(''.join(content))
+        except:
+            raise ParseException("Could not parse json.", ''.join(content))
```

Comment:
@psachin - Would you please replace the `try ... except ...` part to call the `parse_conent` of the base class `JSONParser`, since this one is a child class of it.
```
super(EngineDBQueryVDSMversion, self).parse_content(content)
```

Reply:
HI @xiangce Sure. However I still want to raise an exception if the content is not JSON parseble.

Reply:
Yes, but the function of the base class does what you want：
https://github.com/RedHatInsights/insights-core/blob/8a1ddf23bd942176779784d11e338380ae1062c5/insights/core/__init__.py#L750-L757

Reply:
@xiangce Done